core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4602,15 +4602,15 @@ components:
         resourceAttributes:
           $ref: '#/components/schemas/ResourceAttributesUnion'
       description: resource description
-    DataRepoSnapshotResourceList:
+    SnapshotListResponse:
       type: object
-      required: [ resources ]
+      required: [ gcpDataRepoSnapshots ]
       properties:
-        resources:
+        gcpDataRepoSnapshots:
           type: array
-          description: A list of snapshot Data Resource definitions from Workspace Manager
+          description: A list of DataRepoSnapshotResources from Workspace Manager
           items:
-            $ref: '#/components/schemas/ResourceDescription'
+            $ref: '#/components/schemas/DataRepoSnapshotResource'
```

Comment:
I think `ResourceDescription` is now unused and could be deleted, which in turn would make `ResourceAttributesUnion` unused and deletable. Fine if you'd rather do this in a follow-on PR along with deleting `SnapshotApiService.resourceListToDataReferenceList`

Reply:
Good point.